pushapkscript/googleplay.py

```diff
@@ -42,8 +43,11 @@ def should_commit_transaction(task_payload):
     return task_payload.get('commit', False)
 
 
-def craft_valid_track_values(has_nightly_track):
-    return _DEFAULT_TRACK_VALUES + (['nightly'] if has_nightly_track else [])
+def craft_valid_track_values(require_track, has_nightly_track):
+    if require_track:
```

Comment:
Okay to keep this solution for now. I'd love to migrate everything under a single way of defining tracks.